subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/buildcache/TaskOutputCachingJavaPerformanceTest.groovy

```diff
@@ -159,7 +236,7 @@ class TaskOutputCachingJavaPerformanceTest extends AbstractTaskOutputCachingPerf
         result.assertCurrentVersionHasNotRegressed()
 
         where:
-        testScenario << [scenarios, [true, false]].combinations()
+        testScenario << [[LARGE_JAVA_MULTI_PROJECT], [true, false]].combinations() + [LARGE_MONOLITHIC_JAVA_PROJECT, false]
```

Comment:
monolithic project with parallel true scenario is removed because it doesn't make any sense.